src/controller/bankrupt/bankrupt-officer.controller.ts

```diff
@@ -4,15 +4,15 @@ import { logger, formattingOfficersInfo } from '../../utils';
 
 export default async (req: Request, res: Response, next: NextFunction): Promise<void> => {
   try {
-    const bankruptOfficer = await fetchBankruptOfficer(req.params?.id);
-    
-    if (!bankruptOfficer.error) {
-      const officer = (bankruptOfficer.data) ? formattingOfficersInfo([bankruptOfficer.data])[0] : {};
+    const bankruptOfficer = await fetchBankruptOfficer(req.session, req.params?.id);
+
+    if (bankruptOfficer.httpStatusCode === 200){
+      const officer = (bankruptOfficer.resource) ? formattingOfficersInfo([bankruptOfficer.resource])[0] : {};
```

Comment:
why did we replace the `.data` with the `.resource`?

Reply:
The `resource` object is returned from the SDK